python/debugging/diagnostics/SinksDatabases.ql

```diff
@@ -0,0 +1,16 @@
+/**
+ * @name Database Sinks Diagnostic
+ * @id ghsl/diagnostics/database-sinks
+ * @description List all database sinks
+ * @kind diagnostic
```

Comment:
What is `diagnostic` for? Is it presented in a special way?

Reply:
Yes, they are added to the SARIF in a particular way